libutils/Makefile.am

```diff
@@ -79,6 +79,7 @@ libutils_la_SOURCES = \
 	version_comparison.c version_comparison.h \
 	writer.c writer.h \
 	xml_writer.c xml_writer.h \
+	fsattrs.c fsattrs.h \
 	glob_lib.c glob_lib.h
```

Comment:
@larsewi looks like this list was alphabetic before glob_lib and fsattrs, can you sort it again?